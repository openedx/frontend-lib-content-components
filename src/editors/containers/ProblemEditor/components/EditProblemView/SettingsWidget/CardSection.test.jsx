import { shallow } from '@edx/react-unit-test-utils';
import CardSection from './CardSection';

describe('CardSection', () => {
  test('open', () => {
    expect(shallow(<CardSection summary="summary" isCardCollapsibleOpen><h1>Section Text</h1></CardSection>).snapshot).toMatchSnapshot();
  });

  test('closed', () => {
    expect(
      shallow(<CardSection isCardCollapsibleOpen={false}><h1>Section Text</h1></CardSection>).snapshot,
    ).toMatchSnapshot();
  });
});
