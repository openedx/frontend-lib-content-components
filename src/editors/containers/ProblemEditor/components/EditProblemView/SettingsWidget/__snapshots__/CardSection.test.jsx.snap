// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSection closed 1`] = `null`;

exports[`CardSection open 1`] = `
<Card.Section
  className="pt-0"
>
  <Advanced
    open={false}
  >
    <Body
      className="collapsible-body"
    >
      <span
        className="small text-primary-500"
      >
        summary
      </span>
    </Body>
  </Advanced>
  <Advanced
    open={true}
  >
    <Body
      className="collapsible-body text-primary-500 x-small"
    >
      <h1>
        Section Text
      </h1>
    </Body>
  </Advanced>
</Card.Section>
`;
